### PR TITLE
Add tests and minor fixes for collapsible-section

### PR DESCRIPTION
**Pull request by adrianschmidt**
To test the new tests, I would recommend looking at the test description, then breaking the code, so that it no longer does what the test description says, then check that the test fails.

So, for something like `limel-collapsible-section > with a header and body > when clicking the header > opens`, you might want to comment out the code inside the `onClick` function. Then check that that specific test fails in the expected way. If other tests fail (which some probably will), check that it makes sense that they fail because of the change you made. Then undo the change, and repeat for the next test 🙂

## Review:
- [ ] Commits are [atomic](https://seesparkbox.com/foundry/atomic_commits_with_git)
- [ ] Commits have the correct *type* for the changes made
- [ ] Commits with *breaking changes* are marked as such

### Browsers tested:
(Check any that applies, it's ok to leave boxes unchecked if testing something didn't seem relevant.)

Windows:
- [ ] Chrome
- [ ] Edge
- [ ] Firefox

Linux:
- [ ] Chrome
- [ ] Firefox

macOS:
- [ ] Chrome
- [ ] Firefox
- [ ] Safari

Mobile:
- [ ] Chrome on Android
- [ ] iOS

### CODE REVIEW

File: src/util/e2eClickElement.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import { E2EElement, E2EPage } from '@stencil/core/testing';
+
+/* eslint-disable no-underscore-dangle */
+export async function clickElement(page: E2EPage, elements: E2EElement[]) {
```
Comment (jgroth) [inline src/util/e2eClickElement.ts:4]: What is `elements` here? Given the name of the function I would expect just a single element, what are the other elements used for? :slightly_smiling_face: :thinking:

**Review by jgroth — COMMENTED** [review]
Great with some new tests! :partying_face: 
Looks good, just had a small question

File: src/util/e2eClickElement.ts (hunk 1 above)
Comment (adrianschmidt) [reply src/util/e2eClickElement.ts:4]: Yeah, I think I need to add a docblock here…

If you just send in a single element, it clicks that. But if your element is inside a shadowDOM, you need to give it the respective shadowRoot hosts too…

I'll see what I can come up with to make it clearer 👍

Comment (adrianschmidt) [reply src/util/e2eClickElement.ts:4]: Ah… I think this was a completely redundant invention by me… 🤦‍♂️

image

**Review by jgroth — DISMISSED** [review]

**Comment by adrianschmidt** [comment]
Squashed 🙂

**Review by jgroth — APPROVED** [review]

**Pull request merged**